# The tag id, in the 36h11 family
int16 id
# The tag size in meters, excluding the white border
float64 size
# The tag pose with respect to the object
geometry_msgs/Transform oMt
